### PR TITLE
[OPB-647] Update Observability Pipelines installation docs to use run

### PR DESCRIPTION
**Pull request by binarylogic**

### CODE REVIEW

File: content/en/observability_pipelines/installation.md [hunk 1]
```diff
@@ -66,10 +66,19 @@ $ DD_API_KEY=<DD_API_KEY> DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.
...
 
-4. Start the Worker:
+4. Save your Observability Pipelines configuration:
 
     ```
-    DD_API_KEY=<DD_API_KEY> DD_OP_CONFIG_KEY=<DD_OP_CONFIG_KEY> observability-pipelines-worker 
+    echo "<DD_OP_CONFIG>" > /var/lib/observability-pipelines-worker/observability-pipelines-worker.yaml
+    ```
+
+    Where `DD_OP_CONFIG` are the contents of your Observability Pipeline configuration that you created in the [Observability Pipelines UI].
```
Comment (sghall) [inline content/en/observability_pipelines/installation.md:75]: :two_cents:
```suggestion
    Where `DD_OP_CONFIG` is the contents of your Observability Pipeline configuration that you created in the [Observability Pipelines UI].
```

Comment (sghall) [reply content/en/observability_pipelines/installation.md:75]: Also does `[Observability Pipelines UI]` need a link?

Comment (maycmlee) [reply content/en/observability_pipelines/installation.md:75]: +1 on both comments. Also can we update `contents` to `content`?

Comment (binarylogic) [reply content/en/observability_pipelines/installation.md:75]: Thanks. Done.

**Pull request merged**